tests/test_apply.py

```diff
@@ -1466,6 +1467,424 @@ def test_qubitunitary_rotation_hadamard(self, theta, phi, varphi, shots, qubit_d
         assert np.allclose(var, expected, atol=tolerance, rtol=0)
 
 
+# This block of similar tests coming from pennylane will not fail:
```

Comment:
I added it here as a reference. This was adapted from pennylane tests, and they will not fail, even the ones for tensors and samples (the ones failing for us now)